.ibm/pipelines/utils.sh

```diff
@@ -399,6 +399,8 @@ apply_yaml_files() {
     local RHDH_BASE_URL=$(echo -n "https://${release_name}-backstage-${project}.${K8S_CLUSTER_ROUTER_BASE}" | base64 | tr -d '\n')
     if [[ "$JOB_NAME" == *aks* || "$JOB_NAME" == *gke*  ]]; then
       RHDH_BASE_URL=$(echo -n "https://${K8S_CLUSTER_ROUTER_BASE}" | base64 | tr -d '\n')
+    elif [[ "$JOB_NAME" == *operator* ]]; then
```

Comment:
Please take a look at this comment https://github.com/janus-idp/backstage-showcase/pull/2073/files#r1880119929